packages/rocketchat-logger/client/views/viewLogs.coffee

```diff
@@ -96,3 +97,8 @@ Template.viewLogs.onRendered ->
 		Meteor.setTimeout ->
 			template.checkIfScrollIsAtBottom()
 		, 2000
+
+	wrapper.addEventListener 'scroll', ->
+		template.atBottom = false
+		Meteor.defer ->
+			template.checkIfScrollIsAtBottom()
```

Comment:
You could say the same thing about the wheel/mousewheel events. With the current event listeners, if a frictionless scrollwheel is used, then the check will also happen rapidly until the wheel slows down.

With a throttle, there would be chance that `atBottom` will not be set back to `true`, causing a new issue where the client does not autoscroll when new messages are received.
I already know this would cause issues for me. Sometimes my mouse's scroll wheel misfires causing it to scroll up and then back down very quickly.